calico/_includes/release-notes/v3.23.0-release-notes.md

```diff
@@ -0,0 +1,145 @@
+ - Switch to protobuf for Kubernetes API calls [calico #5409](https://github.com/projectcalico/calico/pull/5409) (@l1b0k)
+ - Added 'readiness_gates' config to the CNI plugin, which will make it wait for the configured endpoints to be ready before launching the pod. [calico #5399](https://github.com/projectcalico/calico/pull/5399) (@VivekThrivikraman-est)
+ - Wireguard PersistentKeepAlive enabled by default and wireguardKeepAlive parameter added to control PersistentKeepAlive option. [calico #5338](https://github.com/projectcalico/calico/pull/5338) (@makhov)
+ - Allow configuration of bind mode via the BGPConfiguration API [calico #5279](https://github.com/projectcalico/calico/pull/5279) (@hanamantagoudvk)
+
+##### eBPF dataplane
+ - ebpf - workloads no longer adjust pmtu when FIB fails due to mtu check - the check is not needed anymore. [calico #5937](https://github.com/projectcalico/calico/pull/5937) (@sridhartigera)
+ - ebpf: Add feature detection and also feature detection override in bpf data plane, similar to iptables. This allows to govern whether IPIP device is L3 or not. [calico #5929](https://github.com/projectcalico/calico/pull/5929) (@mazdakn)
+ - ebpf - strict RPF enforced on workloads fully in BPF regardless of the workloads interface settings. [calico #5923](https://github.com/projectcalico/calico/pull/5923) (@tomastigera)
+ - ebpf: Faster forwarding on CO-RE enabled kernels 5.7+ [calico #5822](https://github.com/projectcalico/calico/pull/5822) (@tomastigera)
+ - Felix loads CO-RE enabled bpf programs in CO-RE capable kernels. That alows us to use  more recent bpf feature where available. Starts with proper testing whether a packet is GSO which should address some MTU related nodeport issues when the host is a KVM (use virtio) VM. [calico #5783](https://github.com/projectcalico/calico/pull/5783) (@sridhartigera)
+ - ebpf: pod can connect to self when CTLB is disabled. [calico #5393](https://github.com/projectcalico/calico/pull/5393) (@tomastigera)
+ - FELIX_BPFEnforceRPF=Disabled/Strict (Strict by default) enforces strict RPF regardless of device settings in BPF mode. [calico #5342](https://github.com/projectcalico/calico/pull/5342) (@tomastigera)
+
+##### helm chart
+
+ - The tigera-operator helm chart now requires the user pre-create the `tigera-operator` namespace, or pass the `--create-namespace` option to `helm install`. Additionally, there are specific upgrade steps that MUST be performed when upgrading a helm install to Calico v3.23. Please see [the helm upgrade documentation](https://projectcalico.docs.tigera.io/maintenance/kubernetes-upgrade#upgrading-an-installation-that-was-installed-using-helm) for instructions. [calico #5649](https://github.com/projectcalico/calico/pull/5649) (@caseydavenport)
```

Comment:
Wonder if we should have a "Upgrade instructions" section or something that calls out this as well as the CSI work (I think those are the only two with any upgrade steps?) 

Probably worth updating our template to have a "Breaking changes" or "Require user action" section at the very top for important notices like this.